core/controllers/profile.py

```diff
@@ -532,6 +532,23 @@ def post(self):
 class SiteLanguageHandler(base.BaseHandler):
     """Changes the preferred system language in the user's preferences."""
 
+    Supported_Languages = [
+        'en', 'pt-br', 'ar', 'hi', 'es', 'bn', 'fr', 'id',
+        'uk', 'sk', 'nl', 'kab', 'vi', 'tr', 'zh-hans', 'zh-hant'
+    ]
```

Comment:
Doesn't this list already exists elsewhere?

Reply:
![supportLanguages](https://user-images.githubusercontent.com/78922463/153416213-ff6d0cd6-36ba-47fb-a048-4dadcc3e8a6b.png)
There is a list but since we just need to use language code as the string value, I just used the same list and edited as how we need in this situation. And yes, I have cross checked that no language is missing from this list.